demos/googleVisionAI/main.yml

```diff
@@ -36,8 +36,8 @@ services:
         constraints: [node.labels.type != head]
     depends_on:
       - yserver
-    command: sh -c "googleVisionAI" 
- 
+    command: sh -c "googleVisionAI"
```

Comment:
are we 100% sure the dependency on `yserver` service is enough? Shouldn't we include also the wait here?